docs/docs/components/lifecycle-operator/webhook.md

```diff
@@ -0,0 +1,70 @@
+---
+comments: true
+---
+
+# Keptn Webhooks
+
+Keptn uses
+[Admission Webhooks](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#what-are-admission-webhooks)
```

Comment:
shouldn't it be mutating wehbooks? i don't think we have admission webhooks

Reply:
admission webhook can be either validating or mutating webhook, so I guess it's ok here..

Reply:
wow, TIL! 😄